### PR TITLE
prime numbers in a range

### PR DESCRIPTION
**Pull request by MadhuBeesetty**

### CODE REVIEW

File: test/41.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    }
+    if( i == a.length-1 ){
+      console.log("your programme is good");
+      return ;
+  }
+    return test1(i+1);
+  }
+  test1();
+}
+test(100);
```
Comment (allopez7) [inline test/41.js:21]: add two more tests.

File: solutions/41.js [hunk 2]
```diff
@@ -1,6 +1,5 @@
 const prime = (limit,a=2,b=[])=>{
   if(a==limit){
-    console.log(b);
     return b;
   }
   const prime1 = (i=2)=>{
```
Comment (allopez7) [inline solutions/41.js]: There is no need for this function.

File: solutions/41.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+const prime = (limit,a=2,b=[])=>{
+  if(a==limit){
+    return b;
+  }
+  const prime1 = (i=2)=>{
```
Comment (seemcat) [inline solutions/41.js:5]: Why not use the previous function you created?

**Review by allopez7 — APPROVED** [review]

File: solutions/41.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    if(a%i==0){
+      if(a==i){
```
Comment (songz) [inline solutions/41.js]: this looks a little ugly here. This is better:
```
if( a !== i) return;
b.push(a);
```

seems like you are not returning any value. In that case, you don't even need to use return at all. The function automatically (by default) returns nothing

**Review by seemcat — CHANGES_REQUESTED** [review]

File: solutions/41.js (hunk 4 above)
Comment (MadhuBeesetty) [reply solutions/41.js]: thank you song, made the changes

**Pull request merged**